backend/routes/parking.py

```
@@ -0,0 +1,140 @@
+    - **longitude**: Longitude coordinate (-180 to 180)
+    - **radius_meters**: Search radius in meters (10-5000, default 100)
+    
+    Returns nearby parking signs and public parking facilities.
+    """
+    log.info(f"Parking search request: {request.latitude}, {request.longitude}")
+    
+    try:
+        result = await parking_service.search_parking(request)
+        
+        log.info(f"Search completed: {len(result.parking_sign_results)} signs, {len(result.public_parking_results)} facilities")
+        return result
```

Ruff (0.13.1)

Consider moving this statement to an `else` block

(TRY300)